### ISSUE TITLE
Typo in readme.md

### ISSUE THREAD

**Issue by youmad** [VasekPurchart/Doctrine-Date-Time-Immutable-Types-Bundle#19]
```php
<?php

// created date can no longer be modified from outside
var_dump($logRow->getCreatedDate()); // 2015-01-01 00:00:00
$logRow->getCreatedDate()->modify('+14 days');
var_dump($logRow->getCreatedDate()); // 2015-01-11 00:00:00
```
The date has changed in comment.

**Comment by VasekPurchart** [comment]
Thanks for noticing, this is corrected in 29768df and also in the underlying library.

**Comment by rgazelot** [comment]
You're welcome :)
